kubejs/server_scripts/base/tags/items/emendatusenigmatica/gold.js

```diff
@@ -0,0 +1 @@
+// TODO: Delete next breaking update
```

Comment:
oh I'm wrong, this file is actually new :P please delete. Sorry!

Reply:
No no no! I was wrong, it is in the release 😅 I just had it modified from the start iirc and forgot that it also was in the previous PR.
![obraz](https://github.com/EnigmaticaModpacks/Enigmatica9/assets/60540476/b777aebe-7564-4942-aa73-52e4d25bd99f)

Reply:
Oh, I misinterpreted GitHub 😅